src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -37,6 +37,7 @@ uct_rc_verbs_ep_post_send(uct_rc_verbs_iface_t* iface, uct_rc_verbs_ep_t* ep,
 
     ucs_assertv(ep->qp->state == IBV_QPS_RTS, "QP 0x%x state is %d",
                 ep->qp->qp_num, ep->qp->state);
+    uct_rc_txqp_check_not_failed(&ep->super.txqp);
```

Comment:
given the line above, we need the check only for mlx5 qps
so can move flags to txwq

Reply:
thanks for catching this, done